src/DetailsView/reports/components/report-head-v2.tsx

```diff
@@ -8,7 +8,7 @@ import * as reportStyles from '../automated-checks-report.styles';
 export const ReportHeadV2 = NamedSFC('ReportHead', () => {
     return (
         <head>
-            <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
+            <meta httpEquiv="Content-Type" content="text/html;charset=utf-8" />
```

Comment:
This was causing a console error.

Reply:
wasn't this change made on one of Haonan's PR? seems duplicated... I guess it's fine as long as we don't see conflicts.

Reply:
I added this previously to fix the special character issue in Edge, but I guess I messed up the naming of parameter.

Reply:
Sure... what I'm saying is this change is already on master... so why making it again? is one more file reviewers need to go through... also... this change have nothing to do with the scan details section accessibility... so... hard to figure out

Reply:
It was causing a console error even on master. Doesn't fail or anything, just to fix that red error! :)